lxqt-sudo/lxqt-sudo-pt.ts

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.0" language="pt_PT">
+<context>
+    <name>PasswordDialog</name>
+    <message>
+        <location filename="../../passworddialog.ui" line="20"/>
+        <source>LXQt sudo</source>
+        <translation>LXQt sudo</translation>
+    </message>
+    <message>
+        <location filename="../../passworddialog.ui" line="77"/>
+        <source>Command:</source>
+        <translation>Comando:</translation>
+    </message>
+    <message>
+        <location filename="../../passworddialog.ui" line="84"/>
+        <source>Password:</source>
+        <translation>Palavra-passe:</translation>
+    </message>
+    <message>
+        <location filename="../../passworddialog.cpp" line="43"/>
+        <source>&lt;b&gt;%1&lt;/b&gt; needs administrative privileges.
+Please enter your password.</source>
+        <translation>&lt;b&gt;%1&lt;/b&gt; necessita de ser executado como administrador.
+Por favor digite a sua palavra-passe.</translation>
+    </message>
+    <message>
+        <location filename="../../passworddialog.cpp" line="54"/>
+        <source>Attempt #%1</source>
+        <translation>Tentativa #%1</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../../sudo.cpp" line="63"/>
+        <source>Usage: %1 option [command [arguments...]]
+
+GUI frontend for %2/%3
+
+Arguments:
+  option:
+    -h|--help      Print this help.
+    -v|--version   Print version information.
+    -s|--su        Use %3(1) as backend.
+    -d|--sudo      Use %2(8) as backend.
+  command          Command to run.
+  arguments        Optional arguments for command.
+
+</source>
+        <translation>Utilizaçãoe: %1 opção [comando [argumentos...]]
```

Comment:
Hi Paulo

Thanks for the notice. Since I´m not expert in working with git and I already have a new set of files for commit, could you please commit this one as is and in the next set I will fix this one?